### PR TITLE
[v0.31] release / release notes (PR 1 of 3)

### PR DESCRIPTION
**Pull request by dennisocana**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:

Release notes for version 0.31.0 release.

image
image
image
image
image
image

### CODE REVIEW

**Review by carlolm — CHANGES_REQUESTED** [review]
@dennisocana should include mention of crypto.com connector

Blog URL will be: https://hummingbot.io/blog/2020-09-hummingbot-partners-with-crypto-com/

**Comment by dennisocana** [comment]
@carlolm just added docs for crypto.com connector and included the announcement in the release notes.

**Review by carlolm — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ We created hummingbot to promote **decentralized market-making**: enabling membe
 | <img src="documentation/docs/assets/img/kucoin_logo.png" alt="KuCoin" width="90" /> | kucoin | [KuCoin](https://www.kucoin.com/) | 1 | [API](https://docs.kucoin.com/#general) |
```
Comment (carlolm) [inline README.md:23]: Should the Bittrex api be updated to 3? @dennisocana

Comment (dennisocana) [reply README.md:23]: @carlolm yes, good catch. I will make the change now.

**Review by Nullably — APPROVED** [review]
LGTM

**Pull request merged**